Tests/XMLCoderTests/Minimal/StringTests.swift

```diff
@@ -25,15 +25,6 @@ class StringTests: XCTestCase {
         ("foobar", "foobar"),
     ]
 
-    func testMissing() {
-        let decoder = XMLDecoder()
-
-        let xmlString = "<container />"
-        let xmlData = xmlString.data(using: .utf8)!
-
-        XCTAssertThrowsError(try decoder.decode(Container.self, from: xmlData))
-    }
-
```

Comment:
Does this test pass? If so, I'd prefer to keep it in the test suite

Reply:
I've played around with this, and it does work if the element is renamed to, say, `notValue`, but I'm encountering an ambiguity with the treatment of the `value` intrinsic. Would you be content with this if we pushed through the proposed change from `value` to `xmlValue` as the labeling of the intrinsic?

Reply:
(or even limiting the intrinsic to `""`)

Reply:
`""` and `"xmlValue"` are both fine, `"xmlValue"` (or everything that starts with "xml" for that matter) is not a valid element or attribute, at least according to my understanding of the XML standard:

>Names beginning with the string "xml", or with any string which would match (('X'|'x') ('M'|'m') ('L'|'l')), are reserved for standardization in this or future versions of this specification.

Reply:
On the other hand, isn't decoding the value of `<container />` element to an empty string is the expected behaviour here? I think I also relied on this in [CoreXLSX](https://github.com/MaxDesiatov/CoreXLX) in some of its behavior if I remember correctly, but this needs to be confirmed.

Reply:
This might seem a bit self-centered, but if it doesn't break CoreXLSX, I'm probably fine with the change, hope MusicXML test suite would pass as well. Just wondering how many other apps and libraries we could break with this change and what would be possible workarounds in that case.

Reply:
Hoping we can perhaps circumvent these concerns by changing the use of the `"value"` coding key, per my latest PR! It does mean downstream users would have to introduce `= ""` additions, but since this change is probably on the way anyway, I wonder if now is as good a time as any...?

Reply:
@bwetherfield Will this test be added back?